usaspending_api/references/serializers_v2.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+from usaspending_api.references.models import (
+    Agency, Cfda, LegalEntity, Location, ObjectClass, OfficeAgency,
+    RefProgramActivity, SubtierAgency, ToptierAgency, LegalEntityOfficers, Definition)
+from usaspending_api.common.serializers import LimitableSerializer
+
+
+class AgencyV2Serializer(serializers.Serializer):
+
+    agency_name = serializers.CharField()
+    # active_fiscal_year = serializers.IntegerField()
```

Comment:
We made the obligations a `DecimalField`, should we stick with this instead of the line below it?